techniques/system/server-common/1.0/bootstrapRudderServer.cf

```diff
@@ -0,0 +1,16 @@
+# Used only for bootstrapping Rudder server
+bundle agent bootstrap_server_rudder {
```

Comment:
when is this called ?

Reply:
by good ol' rudder-init

Reply:
see https://github.com/Normation/rudder-packages/pull/2528